packages/jest-config/src/resolveConfigPath.ts

```diff
@@ -77,8 +77,8 @@ const resolveConfigPathByTraversing = (
     configFiles.push(packageJson);
   }
 
-  if (!skipMultipleConfigWarning && configFiles.length > 1) {
-    console.warn(makeMultipleConfigsWarning(configFiles));
+  if (!skipMultipleConfigError && configFiles.length > 1) {
+    throw new Error(makeMultipleConfigsErrorMessage(configFiles));
```

Comment:
```suggestion
    throw new ValidationError(makeMultipleConfigsErrorMessage(configFiles));
```

`import {ValidationError} from 'jest-validate';`

that should strip the stack trace I mentioned above as well as well

Reply:
okay, done.